src/utils/daos_hdlr.c

```diff
@@ -848,6 +850,50 @@ cont_get_prop_hdlr(struct cmd_args_s *ap)
 	return rc;
 }
 
+int
+cont_set_prop_hdlr(struct cmd_args_s *ap)
+{
+	int		rc;
+	daos_prop_t	*prop = NULL;
+
+	if (ap->name == NULL || ap->value_str == NULL) {
+		fprintf(stderr, "--name and --value params required\n");
+		return -DER_INVAL;
+	}
+
+	/* Only set one at a time */
+	prop = daos_prop_alloc(1);
+	if (prop == NULL)
+		return -DER_NOMEM;
+
+	/* Only label supported for now */
+	if (strncmp(ap->name, "LABEL", MAX_PROP_NAME_LEN) == 0) {
```

Comment:
I noticed in daos_parse_property() function it does strcmp(name, "label") lower case. Should it be the same, or maybe even case indepedent?

Reply:
Ah, good catch. I'll match what comes out of get-prop for now. If we want to make it case-insensitive, that could be a future improvement